packages/api/src/command/medical/document/document-webhook.ts

```diff
@@ -86,8 +86,9 @@ export const processPatientDocumentRequest = async (
         requestId ? { requestId } : undefined,
         metadata
       );
+      log(`WH Sent successfully`);
```

Comment:
I'd remove this unless you're sure we can only reach this part of the code if the WH gets a 200...